includes/compatibility/class-compat.php

```diff
@@ -108,5 +113,11 @@ public function _fix_pagination_issue( $clauses, $query ) {
 
 		return $clauses;
 	}
-
+	public function wp_fusion_skip_auto_login( $skip_auto_login ) {
+		if ( ! empty( $_REQUEST['action'] ) && ! empty( $_REQUEST['handler'] )
+			&& $_REQUEST['action'] === 'wpbdp_ajax' && $_REQUEST['handler'] === 'checkout__load_gateway' ) {
```

Comment:
Let's use the wpbdp_get_var function here too to get all $_REQUEST params for security.